### ISSUE TITLE
incorrect warning from readNWISqw

### ISSUE THREAD

**Issue by dlorenz-usgs** [USGS-R/dataRetrieval#95]
I got a warning from

Well <- "364200119420001"
Pcodes <-c("00010", "00095", "00300", "00400")

TestWellQW <- readNWISqw(Well, Pcodes, expanded=FALSE, reshape=TRUE)
 It was:
Warning message:
In readNWISqw(Well, Pcodes, expanded = FALSE, reshape = TRUE) :
  Reshape can only be used with expanded data. Reshape request will be ignored.

But the data were reshaped and it does in fact make sense to reshape the data when expanded=FALSE (maybe even more so than when expanded is TRUE).

**Comment by ldecicco-USGS** [comment]
The data may have been reshaped by NWIS, but not `dataRetrieval`. The data coming back from NWIS is in a 'wide' format - the raw column headers are: r00010	p00010	r00095	p00095	r00300	p00300	r00400	p00400.
Are you suggesting to reshape to a long format in the non-expanded request?

Here's the raw data url:
http://nwis.waterdata.usgs.gov/nwis/qwdata?search_site_no=364200119420001&search_site_no_match_type=exact&multiple_parameter_cds=00010,00095,00300,00400&param_cd_operator=OR&list_of_search_criteria=search_site_no,multiple_parameter_cds&group_key=NONE&sitefile_output_format=html_table&column_name=agency_cd&column_name=site_no&column_name=station_nm&inventory_output=0&rdb_inventory_output=file&TZoutput=0&pm_cd_compare=Greater%20than&radio_parm_cds=previous_parm_cds&qw_attributes=0&format=rdb&rdb_qw_attributes=0&date_format=YYYY-MM-DD&rdb_compression=value&qw_sample_wide=separated_wide

```r
Well <- "364200119420001"
Pcodes <-c("00010", "00095", "00300", "00400")

TestWellQW <- readNWISqw(Well, Pcodes, expanded=FALSE, reshape=TRUE)
attr(TestWellQW , "url")
```

**Comment by dlorenz-usgs** [comment]
But the documentation for reshape states

logical, reshape the data. If TRUE, then return a wide data frame with all water-quality in a single row for each sample. If FALSE (default), then return a long data frame with each water-quality result in a single row.

I want a wide dataset without expanded fields. maybe what needs to be changed is the documentation. It seems from your comments that the data come back in wide format is expanded is FALSE regardless of the value for reshape.  

make sense?

**Comment by ldecicco-USGS** [comment]
Yup - that's exactly it. I'll change the wording in the help file. Ideally, I'd like to not change the actual arguments, so how's this:

logical, reshape the *expanded* data. If TRUE, then return a wide data frame with all water-quality in a single row for each sample. If FALSE (default), then return a long data frame with each water-quality result in a single row. *This argument is only applicable to expanded data. Data that uses `expanded=FALSE` is automatically returned in the wide format.*

OK?

**Comment by dlorenz-usgs** [comment]
I think that works, the term expanded data makes sense from the definition of the argument expanded. But I would rephrase the second sentence:
Data requested using expanded=FALSE is always returned in the wide format.

**Comment by ldecicco-USGS** [comment]
Fixing it right now.

**Comment by ldecicco-USGS** [comment]
Trying something with git...
